source/Playnite/Localization/LocSource.xaml

```diff
@@ -402,6 +413,8 @@ If disabled, background artwork is downloaded only when first needed, using less
     <sys:String x:Key="LOCSettingsShowMediaSizeWarning">Display warning when assigning too large game media</sys:String>
     <sys:String x:Key="LOCOpenDirectoryCommand">Folder open command</sys:String>
     <sys:String x:Key="LOCSettingsPreferredAgeRatingOrg">Preferred age rating organization</sys:String>
+    <sys:String x:Key="LOCSettingsScanLibInstallSizeOnLibUpdate">Scan modified install size of games on library update</sys:String>
+    <sys:String x:Key="LOCSettingsScanLibInstallSizeOnLibUpdateTooltip">Detects if the files of a game have been modified since the last library update and scans them to get the updated install size</sys:String>

```

Comment:
This message should be changed since you now scan only after last specific game calculation date. It's not tied to library update date anymore.

Reply:
Done in https://github.com/JosefNemec/Playnite/pull/3017/commits/ee21d0c395bf95be8721ba98286ea8dac44576e9